examples/manila-csi-plugin/helm-deployment/values.yaml

```diff
@@ -21,7 +21,7 @@ csimanila:
   # nodeAZ: "$(curl http://169.254.169.254/openstack/latest/meta_data.json | jq -r .availability_zone)"
   # Image spec
   image:
-    repository: manila-csi-plugin
+    repository: k8scloudprovider/manila-csi-plugin-amd64
```

Comment:
@ramineni I guess `k8scloudprovider/manila-csi-plugin` is still valid? That's probably more preferable than specifying the architecture explicitly with `manila-csi-plugin-amd64`...